awx/main/tasks.py

```diff
@@ -1910,6 +1910,7 @@ def build_private_data(self, inventory_update, **kwargs):
             group_patterns = '[]'
             group_prefix = 'foreman_'
             want_hostcollections = 'False'
+            want_ssh_host = 'False'
```

Comment:
This is default value?

Reply:
It's given in the example ini file as hard-coded to False.

You have to check the actual code to find the true default behavior:

https://github.com/ansible/ansible/blob/devel/contrib/inventory/foreman.py#L121

It does appear to set it to False.

I would still prefer to not include options we want to obtain the defaults for, but that ship has sailed.